Makefile

```diff
@@ -30,6 +30,17 @@ test: $(SOURCES)
 lint: $(SOURCES)
 	$(DEPSPATH)/golangci-lint run
 
+.PHONY: mdlint
+# Install it locally: https://github.com/igorshubovych/markdownlint-cli#installation
+# Or see `mdlint-ctr` below or https://github.com/DavidAnson/markdownlint#related.
+mdlint:
+	markdownlint '**/*.md'
+
+.PHONY: mdlint-ctr
+# If markdownlint is not installed, you can run markdownlint within a container.
+mdlint-ctr:
+	finch run --rm -v "$(shell pwd):/repo:rw" -w /repo avtodev/markdown-lint:v1 '**/*.md'
```

Comment:
Probably should be docker or nerd since development would be for the Linux platform.

Reply:
Ah you are right. We can use Docker for now.